### PR TITLE
feat(oui-typography): add modifier to icon link

### PR DESCRIPTION
**Pull request by whispyy**

### CODE REVIEW

File: packages/oui-typography/README.md [hunk 1]
```diff
@@ -93,6 +93,15 @@ oui-typography is a package which provides tools as constants and mixins for man
 </div>
 ```
 
+```html:preview
+<div>
+  <a href="#" class="oui-link oui-link_external">
```
Comment (neolitec) [inline packages/oui-typography/README.md]: I don't like the idea to create a modifier to resize a specific icon.
It could be any icon in all the icons.

Comment (whispyy) [reply packages/oui-typography/README.md]: In fact I'd like to know if it's correct to include the icon directly in the modifier. Cause, I hadn't be able to see it in ui-kit ?

**Comment by whispyy** [comment]
As discussed with @neolitec, I should just add the possibility to set and icon in a link. If I want to include the icon "properly" I have no possibility to set it in the ui-kit. So I'll have to make a component to do it.

File: packages/oui-typography/typography.less [hunk 2]
```diff
@@ -10,7 +10,13 @@
 }
 
 .oui-link {
+  @oui-icon-selector: oui-icon;
```
[the remark this answers is not in the document]
Comment (neolitec) [reply packages/oui-typography/typography.less]: 👍

File: packages/oui-typography/README.md [hunk 3]
```diff
@@ -93,6 +93,15 @@ oui-typography is a package which provides tools as constants and mixins for man
 </div>
 ```
 
+```html:preview
+<div>
+  <a href="#" class="oui-link oui-link_icon">
+    The quick brown fox jumps over the lazy dog
+    <i class="oui-icon oui-icon-external_link" aria-hidden="true"></i>
```
Comment (neolitec) [inline packages/oui-typography/README.md]: What a poor idea to name it `external_link`... It's not BEM compliant :(

**Pull request merged**